azext_edge/edge/providers/check/deviceregistry.py

```diff
@@ -122,17 +122,22 @@ def evaluate_assets(
 
             asset_spec = asset["spec"]
             endpoint_profile_uri = asset_spec.get("assetEndpointProfileUri", "")
+            endpoint = get_resources_by_name(
```

Comment:
Please don't use 'endpoint' to describe endpoint profiles since the term endpoint is overloaded.